src/components/generic/Button/Button.jsx

```diff
@@ -20,16 +21,31 @@ export default function Button({
   className,
   isDisabled,
   onClick,
-  title,
+  tooltip,
   style,
   type,
 }) {
   const cssClasses = webexComponentClasses('button', className, {[`-${type}`]: true});
+  const buttonRef = useRef(null);
 
   return (
-    <button className={cssClasses} disabled={isDisabled} title={title} type="button" onClick={onClick} style={style}>
-      {children}
-    </button>
+    <>
+      <button
+        className={cssClasses}
+        disabled={isDisabled}
+        type="button"
+        onClick={onClick}
+        ref={buttonRef}
+        style={style}
+      >
+        {children}
+      </button>
+      {(buttonRef.current && tooltip) && (
```

Comment:
the `Tooltip` component could handle a falsey target, to make its use easier (and more robust)